### PR TITLE
Update dockerfiles and make compatible with Renovate

### PR DESCRIPTION
**Pull request by DavidSpek**
This PR updates the codeserver-python, jupyter-pytorch, jupyter-scipy, jupyter-tensorflow and rstudio-tidyverse dokerfiles to include the latest packages and make them compatible with Renovate.

/cc @thesuperzapper

### CODE REVIEW

**Comment by DavidSpek** [comment]
Error is due to a memory allocation problem.

/test kubeflow-kubeflow-presubmit

File: components/example-notebook-servers/codeserver-python/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+FROM public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/codeserver:master-434b10ab
 
 USER root
 
 # args - software versions
-ARG CODESERVER_PYTHON_VERSION="2021.3.658691958"
-ARG CONDA_VERSION="4.9.2"
+ARG CODESERVER_PYTHON_VERSION=2021.5.842923320
+ # renovate: datasource=github-tags depName=conda/conda versioning=semver
+ARG CONDA_VERSION=4.10.1
```
Comment (thesuperzapper) [inline components/example-notebook-servers/codeserver-python/Dockerfile:8]: Lets remove the `CONDA_VERSION` arg, as if it is updated independent of `MINIFORGE_VERSION` there will be issues.

Comment (thesuperzapper) [reply components/example-notebook-servers/codeserver-python/Dockerfile:8]: We can use bash variable parameter expansion to remove the `-4` suffix: https://wiki.bash-hackers.org/syntax/pe

Comment (DavidSpek) [reply components/example-notebook-servers/codeserver-python/Dockerfile]: Resolved, using `${MINIFORGE_VERSION:0:-2}` now instead.

**Comment by DavidSpek** [comment]
@kimwnasptd Can you maybe take a quick look through this so we can get it merged and I can continue with the last 2 PRs (1 for the last notebook images, and then 1 to update the configmaps with the new tags).

**Comment by thesuperzapper** [comment]
@DavidSpek looks good, thanks!
/lgtm

**Pull request merged**